parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -522,16 +527,26 @@ pub mod bridging {
 	use xcm_builder::UnpaidRemoteExporter;
 
 	parameter_types! {
-		pub BridgeHubKusamaParaId: u32 = 1002;
+		pub BridgeHubKusamaParaId: u32 = 1013;
```

Comment:
Fine for now - but obviously we can't merge this into upstream cumulus. So we'll need to either introduce a new asset-hub runtime for rococo or change our bridge hub parade